src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/preferences/ExportMailboxRepository.java

```diff
@@ -0,0 +1,51 @@
+package de.dhbw.karlsruhe.students.mailflow.external.infrastructure.preferences;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.MailboxExportRepository;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.exceptions.ExportMailboxException;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.value_objects.ExportableMailbox;
+import de.dhbw.karlsruhe.students.mailflow.external.infrastructure.utils.FileHelper;
+import java.io.File;
+import java.time.LocalDateTime;
+/**
+ * @author seiferla
+ */
+public class ExportMailboxRepository implements MailboxExportRepository {
```

Comment:
```suggestion
public class FileExportableMailboxRepository implements ExportableMailboxRepository {
```
Then you could make it here a bit clearer, that we access the FileSystem instead of for example providing a download link